spec/poros/directions_spec.rb

```diff
@@ -0,0 +1,446 @@
+require 'rails_helper'
+
+RSpec.describe Directions do
+  it 'exists' do
+      attributes = {
```

Comment:
You can totally set your test up this way. If we want to cut down on the line of code, we can create a fixture file where you can paste the `attributes hash` and then just do something like (name your file what you like that makes sense)
```
response = File.read('spec/fixtures/directions_test_data.json')
direction_data = JSON.parse(response, symbolize_names: true)
directions = Directions.new(direction_data)
```